modules/gravatar-hovercards.php

```diff
@@ -25,7 +25,6 @@ function grofiles_hovercards_init() {
 	add_action( 'load-edit-comments.php',      'grofiles_admin_cards' );
 	add_action( 'load-options-discussion.php', 'grofiles_admin_cards_forced' );
 
-	Jetpack::enable_module_configurable( __FILE__ );
```

Comment:
You currently have quite a few Gravatar-related options under Settings > Discussion. If we remove that link, folks will lose that reference.

Reply:
Let's work out whether we want to migrate those settings or not in another issue. It seems to me that gravatar integration priority has waned quite a bit. Because these options are not in Calypso, I think we might be able phase them out and perhaps even remove them.